packages/core/cypress/integration/seo.test.js

```diff
@@ -88,10 +88,13 @@ describe('Product Page Seo', () => {
       .should(($el) => {
         expect($el.attr('content')).to.eq('index,follow')
       })
+
     cy.get('link[rel="canonical"]')
       .should('exist')
-      .should(($link) => {
-        expect($link.attr('href')).to.eq(`${storeUrl}${pages.pdp}`)
+      .and(($link) => {
+        const href = $link.attr('href')
+        const regex = new RegExp(`^${href.split('/')[0]}`)
```

Comment:
I did this because I don't have access the product sku to compare and need to validate if "4k-monitor-99988213/p" contains "4k-monitor"